example/scp.c

```diff
@@ -43,7 +43,11 @@ int main(int argc, char *argv[])
     const char *scppath="/tmp/TEST";
     libssh2_struct_stat fileinfo;
     int rc;
+#if defined(WIN32) & defined(HAVE_STRTOI64)
```

Comment:
I don't get a warning with off_t.  Do you?
What do you think of changing all of this to
```
libssh2_struct_stat_size got=0;
```
and then we get to keep the #if out of here.